tracing/src/main/java/io/airlift/tracing/OpenTelemetryConfig.java

```diff
@@ -1,12 +1,15 @@
 package io.airlift.tracing;
 
 import io.airlift.configuration.Config;
+import jakarta.validation.constraints.Max;
+import jakarta.validation.constraints.Min;
 import jakarta.validation.constraints.NotNull;
 import jakarta.validation.constraints.Pattern;
 
 public class OpenTelemetryConfig
 {
     private String endpoint = "http://localhost:4317";
+    private Double samplingRatio = 1d;
```

Comment:
unless this is nullable, you should make this a `double` instead of the object

Reply:
updated